buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -43,7 +43,9 @@
         "failOnStandardError": "false"
       }
     },
-    {
+    {"PB_FeedReadAccessToken": {
```

Comment:
where did this come from?  it looks like you can delete this.

Reply:
I don't kow wth that came from... Removing